components/Explorer/Explorer.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 import React from 'react';
```

Comment:
that's fine. I think the point of linting though is that eventually you don't need to 'fix' it ever, it's just the style you code in. I think it might help if you had linting in your editor/ide (for example, atom can display linting errors in the file in realtime). But don't worry about it too much right now.